S03-operators/precedence.t

```diff
@@ -213,6 +213,19 @@ is(@c, [1,2,3], "@ = binds looser than ,");
 
 # terminator
 
+# superscript exponentiation
```

Comment:
Could you wrap this chunk in a block and give it its own `$i`. Right now it's re-using the variable used by other test 180 lines prior. It's unclear what its value really is and any changes in those 180 lines may affect the variable, not knowing it's used so far down below.

So, something like:

```perl6
{
    my $i = 2;

    # superscript exponentiation
    #?rakudo 3 skip 'superscript exponent associativity not right yet RT #130414'
    ....rest of new tests..
}
```

Also, since this test doesn't throw, we can fudge it as a `todo` rather than `skip` so that the test harness alerts us when it starts to pass:

```perl6
is(2²**3, 256, "mixed exponent does right associative");
```